cli/src/semgrep/commands/ci.py

```diff
@@ -579,6 +588,18 @@ def ci(
                 progress_bar,
             )
 
+        if internal_ci_scan_results:
```

Comment:
LGTM, but do you understand that this will still send the results to the WebApp? Maybe you should intercept this flag earlier during the scan, like before you send the findings to /results and /complete and just print an exit?

Reply:
yeah that's intended. we're going to be running these with dry-run enabled since we don't want anything to end up in the app